packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/deploy-contract-go-source/deploy-contract-go-source-endpoint-v1.ts

```diff
@@ -117,9 +118,10 @@ export class DeployContractGoSourceEndpointV1 implements IWebServiceEndpoint {
       res.json(resBody);
     } catch (ex) {
       this.log.error(`${fnTag} failed to serve contract deploy request`, ex);
-      res.status(HttpStatus.INTERNAL_SERVER_ERROR);
-      res.statusMessage = ex.message;
-      res.json({ error: ex.stack });
+      res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({
```

Comment:
@rwat17 Please use the `handleRestEndpointException` utility instead of hardcoding the status code.

I won't post this everywhere but please address the same change across the PR